code/EnvironmentChecker.php

```diff
@@ -88,11 +88,18 @@ function canAccess($member = null, $permission = "ADMIN") {
 	function index() {
 		$response = new SS_HTTPResponse;
 		$result = EnvironmentCheckSuite::inst($this->checkSuiteName)->run();
-		
+
+		// output the result as JSON if requested
+		if($this->getRequest()->getVar('format') == 'json') {
```

Comment:
This is how I sort of faked it in the deploynaut api:


		if($this->getRequest()->getExtension() == 'json') {
			return true;
		}
		if(strpos($this->getRequest()->getHeader('Accept'), 'application/json') !== false) {
			return true;
		}